supporting_content/dimensions/commitmentdiscounttype.md

```diff
@@ -0,0 +1,17 @@
+# Column: CommitmentDiscountType
+
+## Documentation
+- Microsoft
+  - Azure:  Understand usage details fields: https://learn.microsoft.com/en-us/azure/cost-management-billing/automate/understand-usage-details-fields
+- GCP
+  - Google Commitment Types:  https://cloud.google.com/docs/cuds#spend_versus_resource_commitments (This is the most similar to our selected implementation)
+- AWS
+  - Amazon: Reservation details - https://docs.aws.amazon.com/cur/latest/userguide/reservation-columns.html
+
+## Discussion Topics
```

Comment:
It would be good to add some notes about our discussions on if this should be a broader 'Discount Type' or specifically a 'Commitment Discount Type' and why we chose to go this direction (and possibly why a general discount type might still be useful.

Reply:
Ok, will add

Reply:
Updated